plugins/powerd/fu-plugin-powerd.c

```diff
@@ -74,6 +74,21 @@ fu_plugin_startup (FuPlugin *plugin, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_plugin_inhibit_suspend (GError **error)
+{
+	g_autofree gchar *lockdir = NULL;
+	g_autofree gchar *inhibitsuspend_filename = NULL;
+        g_autofree gchar *getpid_str = NULL;
+
+	lockdir = fu_common_get_path (FU_PATH_KIND_LOCKDIR);
+	inhibitsuspend_filename =
```

Comment:
Style nit. If you find you've run out of space because of 80-cols, you can intent as follows to line up the arguments with the (

```
	inhibitsuspend_filename = g_build_filename (lockdir,
						    "power_override",
						    "fwupd.lock",
						    NULL);

```

Reply:
I think this may be coming from applying clang-format on https://github.com/fwupd/fwupd/issues/3505

Reply:
Actually is this true? I was hoping that ``BinPackArguments: false`` and ``BinPackParameters: false`` would deal with this.